feature-libs/product-multi-dimensional/base/occ/product-multi-dimensional-base-occ.module.ts

```diff
@@ -4,13 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { provideDefaultConfig } from '@spartacus/core';
 import { defaultOccProductMultidimensionalBaseConfig } from './config/default-occ-product-multi-dimensional-base-config';
```

Comment:
nitpick: should we keep the convention and name this variable as: `defaultOccProductMultiDimensionalBaseConfig`?

Reply:
good catch, thanks, fixed